ruleset/decoders/0310-ssh_decoders.xml

```diff
@@ -165,10 +165,11 @@
   <order>srcip</order>
 </decoder>
 
+<!-- Apr 14 19:28:21 gorilla sshd[31274]: Connection closed by 192.168.1.33 port 883 -->
 <decoder name="ssh-closed">
   <parent>sshd</parent>
-  <prematch>^Connection closed </prematch>
-  <regex offset="after_prematch">^by (\S+)$ port (\S+)</regex>
+  <prematch>^Connection closed by (\S+) port (\S+)$</prematch>
```

Comment:
This `prematch` should be:
```
<prematch>^Connection closed by \S+ port \S+</prematch>
```
because there is no need to group in a prematch.

The end-of-string character is not needed.